dbt/models/iasworld/schema/iasworld.sales.yml

```diff
@@ -42,6 +42,10 @@ sources:
                     - parid
                     - who
                     - wen
+                  config:
+                    where: |
+                      substr(saledt, 1, 4) >= '2011'
+                      AND deactivat IS NULL
```

Comment:
This date filter does not match the typical pattern of `unique-conditions`, but it matches how `unique-conditions` is configured for the rest of the `sales` tests:

https://github.com/ccao-data/data-architecture/blob/2d81055921bde290a794efd116296bcb5d5ccb9a/dbt/models/iasworld/schema/iasworld.sales.yml#L68-L72

Reply:
That's fine. `SALES` is one of the few critical tables that doesn't have a `taxyr` per record, hence the weird filter.